Cogs.Publishers/CsSchemaPublisher.cs

```diff
@@ -141,24 +142,20 @@ public void Publish(CogsModel model)
                     }
                     if (!first && (model.Identification.Contains(prop) || "boolintdoubleulong".Contains(prop.DataTypeName))) { toJsonProperties.Append(","); }
                     var start = "((JObject)json.First).Add(";
-                    if (origDataTypeName != null && !"boolintstringulong".Contains(prop.DataTypeName))
-                    {
-                        newClass.Append("$##[JsonConverter(typeof(SimpleTypeConverter))]");
-                        SimpleToJson(origDataTypeName, prop.Name, start, false, newClass);
-                    }
                     if (model.ReusableDataTypes.Contains(item)) { start = "json.Add("; }
+                    if (model.ItemTypes.Contains(prop.DataType) && !item.IsAbstract) { newClass.Append("$##[JsonConverter(typeof(IIdentifiableConverter))]"); }
                     // if there can be at most one, create an instance variable
                     if (!prop.MaxCardinality.Equals("n") && int.Parse(prop.MaxCardinality) == 1)
                     {
-                        if (model.ItemTypes.Contains(prop.DataType) && !item.IsAbstract) { newClass.Append("$##[JsonConverter(typeof(IIdentifiableConverter))]"); }
-                        newClass.Append("$##public " + prop.DataTypeName + " " + prop.Name + " { get; set; }");
-                        if("boolintdoubleulong".Contains(prop.DataTypeName) || model.Identification.Contains(prop))
+                        if ("boolintdoubleulong".Contains(prop.DataTypeName) || model.Identification.Contains(prop))
```

Comment:
We don't know what datatypes might be created as input. A property named bleu would get hit by this comparison to boolintdoubleulong.

Compare against each simple type name individually.